src/app/clients/form/ClientForm.tsx

```diff
@@ -60,6 +62,7 @@ export interface ClientFields extends Fields {
     numberOfAdults: number;
     children: Person[];
     numberOfChildren: number;
+    listType: ListName | "";
```

Comment:
How difficult would it be to have this as `ListName | null`? In general, null is the closest thing we have to the absence of a value, and here it has the benefit that we can't accidentally do string operations on an empty string which would then lose type information. At display time we'd then write something like `value ?? ""` to convert it to an empty string before display